11- Event KeyCodes/script.js

```diff
@@ -0,0 +1,20 @@
+const insert = document.getElementById('insert')
+
+
+window.addEventListener('keydown', (e) => {
+  insert.innerHTML = `
+  <div class="key">
+  ${e.key = " " ? e.code : e.key}
```

Comment:
Shouldn't this be `${e.key === " " ? e.code : e.key}`? I mean the equal sign after `e.key` should either be double or triple but not one.

Reply:
i already fix it in another commit (Fixing bug commit)

Reply:
sweet, thanks!